include/ceed/hash.h

```diff
@@ -1,103 +0,0 @@
-// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
```

Comment:
Note - do we want to delete these? No harm in keeping them and I'm not sure if anyone using libCEED is using this hashing or not

Reply:
I think this is okay to consider as private, but I think there's a decent chance we'll want to use this code again. BTW, the copyright header is incorrect since the copyright statement from PETSc was supposed to be preserved (`UChicago Argonne, LLC and the PETSc Development Team`). It is also BSD-2-Clause so that part is correct.

Reply:
I would say remove for now and easy enough to restore them back into the repo (as private) once they are needed again? I don't see the benefit in keeping around the files if they aren't used anywhere for now.

Reply:
cool!